test/forge/libs/SortedPriceList.t.sol

```diff
@@ -197,6 +182,9 @@ contract SortedPriceListMock {
     List priceList;
 
     function insert(address validator, uint256 price) external {
+        // This to checks are made in the vote function in our case
+        assert(price != 0);
```

Comment:
It is unreasonable to make some part of the mock different and test this different behaviour